### PR TITLE
Theming: add hint about favicon generation

### PR DESCRIPTION
**Pull request by juliusknorr**
Fix for #7100

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: apps/theming/templates/settings-admin.php [hunk 1]
```diff
@@ -88,5 +88,16 @@
 	<div id="theming-preview">
 		<img src="<?php p($_['logo']); ?>" id="theming-preview-logo" />
 	</div>
+	<div class="theming-hints">
+		<? if (!$_['canThemeIcons']) { ?>
```
Comment (MorrisJobke) [inline apps/theming/templates/settings-admin.php]: Really? `<?` 😉 Let me fix this. 😜

Comment (MorrisJobke) [reply apps/theming/templates/settings-admin.php]: Fixed

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Comment by MorrisJobke** [comment]
I fixed the unit tests 😉

File: apps/theming/templates/settings-admin.php (hunk 1 above)
Comment (juliushaertl) [reply apps/theming/templates/settings-admin.php]: Thanks 😉

**Comment by rullzer** [comment]
Mmm the acceptance tests fail...

**Comment by MorrisJobke** [comment]
> Mmm the acceptance tests fail...

Unrelated also happens on other branches :/

**Pull request merged**